README.md

```diff
@@ -1,2 +1,31 @@
-# Scaleway_global_module
-Terraform module repository to deploy Scaleway infrastructure in a minute
+# Terraform module: deploy your infrastructure in a click
+
+Scaleway's products can be used and deployed through Terraform.
+The purpose of this guide is to help users deploy their infrastructure in a safe and quick way, thanks to a module provided by Scaleway team.
+With this module you will be able to deploy an instance inside a private network, a load balancer, a database, a kapsule cluster, and vpc products: public gateway (DHCP/NAT/SSH Bastion) + private network.
```

Comment:
Could you have a schema of this architecture? How common this architecture is among our customers?